### PR TITLE
api: add comments when admins performs manual actions on task or resolution

### PR DESCRIPTION
**Pull request by rbeuque74**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the new behavior (if this is a feature change)?**
When a human operator modifies a task or a resolution, through a manual action, comment is added to indicate that the task/resolution have been manually modified (audit log like)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: api/handler/resolution.go [hunk 1]
```diff
@@ -250,6 +263,12 @@ func RunResolution(c *gin.Context, in *runResolutionIn) error {
 		return errors.Forbiddenf("You are not allowed to resolve this task")
 	}
 
+	reqUsername := auth.GetIdentity(c)
+	_, err = task.CreateComment(dbp, t, reqUsername, "manually run resolution")
```
Comment (wI2L) [inline api/handler/resolution.go:267]: `manually ran resolution`

File: api/handler/task.go [hunk 2]
```diff
@@ -348,13 +378,32 @@ func WontfixTask(c *gin.Context, in *wontfixTaskIn) error {
...
 		false, // skip validation of task contents, task is dead anyway
 		true,  // do record mark change with last activity timestamp
 	)
+	if err != nil {
+		dbp.Rollback()
+		return err
+	}
+
+	reqUsername := auth.GetIdentity(c)
+	_, err = task.CreateComment(dbp, t, reqUsername, "task changed state to won't fix")
```
Comment (wI2L) [inline api/handler/task.go:397]: `changed task state to WONTFIX`

File: pkg/plugins/builtin/subtask/subtask.go [hunk 3]
```diff
@@ -132,7 +132,7 @@ func exec(stepName string, config interface{}, ctx interface{}) (interface{}, in
 			cfg.Tags = map[string]string{}
 		}
 		cfg.Tags[constants.SubtaskTagParentTaskID] = stepContext.ParentTaskID
-		t, err = taskutils.CreateTask(ctx, dbp, tt, watcherUsernames, resolverUsernames, cfg.Input, nil, "Auto created subtask", cfg.Delay, cfg.Tags)
+		t, err = taskutils.CreateTask(ctx, dbp, tt, watcherUsernames, resolverUsernames, cfg.Input, nil, "Auto created subtask, from task "+stepContext.ParentTaskID, cfg.Delay, cfg.Tags)
```
Comment (wI2L) [inline pkg/plugins/builtin/subtask/subtask.go:135]: s/from task/parent task.

**Review by wI2L — COMMENTED** [review]
LGTM. Proposed comments alternatives.

**Comment by rbeuque74** [comment]
> LGTM. Proposed comments alternatives.

Comments addressed.

**Pull request merged**